api/python/cellxgene_census/src/cellxgene_census/experimental/util/_eager_iter.py

```diff
@@ -16,18 +20,25 @@ def __init__(
         self.iterator = iterator
         self._pool = pool or futures.ThreadPoolExecutor()
         self._own_pool = pool is None
+        self._future: Optional[Future[_T]] = None
+        self.fetch_next()
+
+    def fetch_next(self) -> None:
```

Comment:
this needs to be a private function (underscore) as it should never be called by  outside code.

it also does not do the fetch, but starts the fetch, so I suggest naming it something like _submit_next() or some such.....